diff --git a/plugins/D0RunIICone/ProtoJet.hpp b/plugins/D0RunIICone/ProtoJet.hpp
index 7e0a43f..842d94e 100644
--- a/plugins/D0RunIICone/ProtoJet.hpp
+++ b/plugins/D0RunIICone/ProtoJet.hpp
@@ -195,7 +195,7 @@ void ProtoJet<Item>::NowStable() {
 
 template<class Item>
 void ProtoJet<Item>::print(std::ostream& os) const {
-  os<<"y phi Et = ("<<_y<<", "<<_phi<<", "<<this->_Et<<")"<<std::endl;
+  os<<"y phi pT = ("<<_y<<", "<<_phi<<", "<<this->_pT<<")"<<std::endl;
   os<< " members= " << std::endl;
   typename std::list<const Item*>::const_iterator i;
   for(i = _LItems.begin(); i != _LItems.end(); ++i)
